src/components/Modeler.vue

```diff
@@ -887,59 +885,45 @@ export default {
       };
     });
 
-    this.paper = new dia.Paper({
-      async: true,
-      el: this.$refs.paper,
-      model: this.graph,
-      sorting: 'sorting-approximate',
-      gridSize: 10,
-      drawGrid: true,
-      clickThreshold: 10,
-      perpendicularLinks: true,
-      interactive: this.graph.get('interactiveFunc'),
-      highlighting: {
-        default: { options: { padding: highlightPadding } },
-      },
-    });
-
-    this.paper.translate(168, 20);
+    this.paperManager = PaperManager.factory(this.$refs.paper, this.graph.get('interactiveFunc'), this.graph);
+    this.paper = this.paperManager.paper;
```

Comment:
This reference is still required for now as other components still need the paper object.